src/sentry/api/endpoints/organization_member_index.py

```diff
@@ -19,9 +29,27 @@ class MemberPermission(OrganizationPermission):
     }
 
 
+class OrganizationMemberSerializer(serializers.Serializer):
+    email = serializers.EmailField(max_length=75, required=False)
+    role = serializers.ChoiceField(choices=roles.get_choices(), required=True)
+    teams = ListField(required=False, allow_null=False)
```

Comment:
This is `required=False`. What happens when this endpoint is POSTed no teams?

Same with the `email` field. Unless I'm missing something weird here, these should be required.

Reply:
if you post an empty array of teams, it works fine. (the user is invited but not part of any team until added). agreed re: email

Reply:
it needed to be required false in order to replicate the current behavior